abci/example/kvstore/kvstore.go

```diff
@@ -413,33 +413,33 @@ func isValidatorTx(tx []byte) bool {
 	return strings.HasPrefix(string(tx), ValidatorPrefix)
 }
 
-func parseValidatorTx(tx []byte) ([]byte, int64, error) {
+func parseValidatorTx(tx []byte) (string, []byte, int64, error) {
 	tx = tx[len(ValidatorPrefix):]
 
 	//  get the pubkey and power
-	pubKeyAndPower := strings.Split(string(tx), "!")
-	if len(pubKeyAndPower) != 2 {
-		return nil, 0, fmt.Errorf("expected 'pubkey!power'. Got %v", pubKeyAndPower)
+	typeKeyAndPower := strings.Split(string(tx), "!")
+	if len(typeKeyAndPower) != 3 {
+		return "", nil, 0, fmt.Errorf("expected 'pubkeytype!pubkey!power'. Got %v", typeKeyAndPower)
 	}
-	pubkeyS, powerS := pubKeyAndPower[0], pubKeyAndPower[1]
+	keytype, pubkeyS, powerS := typeKeyAndPower[0], typeKeyAndPower[1], typeKeyAndPower[2]
```

Comment:
Can we change to `keyType` ?

Reply:
Apparently the casing has been kept the same in [v0.38.5](https://github.com/cometbft/cometbft/blob/v0.38.5/abci/example/kvstore/kvstore.go#L425)
cc: @sergio-mena

Reply:
Sorry, I missed this. I'll added to my new (small) PR on CometBFT

Reply:
Resolved